src/v/container/include/container/fragmented_vector.h

```diff
@@ -93,21 +94,13 @@ class fragmented_vector {
     fragmented_vector(fragmented_vector&& other) noexcept {
         *this = std::move(other);
     }
-    fragmented_vector& operator=(fragmented_vector&& other) noexcept {
-        if (this != &other) {
-            this->_size = other._size;
-            this->_capacity = other._capacity;
-            this->_frags = std::move(other._frags);
-            // Move compatibility with std::vector that post move
-            // the vector is empty().
-            other._size = other._capacity = 0;
-            other.update_generation();
-            update_generation();
-        }
-        return *this;
-    }
-    ~fragmented_vector() noexcept = default;
 
+    /**
+     * @brief Create a vector from a begin, end iterator pair.
+     *
+     * This has the same semantics as the corresponding std::vector
+     * constructor.
+     */
     template<typename Iter>
     requires std::input_iterator<Iter>
     fragmented_vector(Iter begin, Iter end)
```

Comment:
nit: would it be possible to change commit message to keep the naming consistent i.e. `fragemented_vector` ?

Reply:
Sure, updated to use `fragmented_vector`.